wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/SectionTableFooter.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireDesign
+
+final class SectionTableFooter: UITableViewHeaderFooterView {
+
+    let footerView = SectionFooterView()
+
+    var titleLabel: UILabel {
```

Comment:
I don't see that we use titleLabel here ?

Reply:
It's used outside of this class. I would like to clean up and organize all of our footers and headers but it's outside of the code of this PR.

Reply:
Looks like `SectionTableFooter` and `SectionFooterView` are the same, right?

Reply:
Not exactly. The `SectionTableFooter` is of type `UITableViewHeaderFooterView` & `SectionFooterView` a `UIView`. To be honest, even before opening the PR I found that setup confusing and that's why I want to simplify it as soon as possible. I think we're trying to do something simple using something complex.